bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/Clip2Thing.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- API v2 device thing -->
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge-api2"/>
+		</supported-bridge-type-refs>
+
+		<label>Hue Device</label>
+		<description>A Hue API v2 device with channels depending on its actual capabilities.</description>
+
+		<channels>
+			<channel id="brightness" typeId="light-brightness"/>
+			<channel id="color" typeId="system.color"/>
+			<channel id="color-temperature" typeId="system.color-temperature"/>
+			<channel id="color-temp-kelvin" typeId="system.color-temperature-abs"/>
+			<channel id="switch" typeId="light-switch"/>
+			<channel id="dynamics" typeId="dynamics"/>
+			<channel id="alert" typeId="alert2"/>
```

Comment:
One thing about type ID naming here: maybe name this `alert-v2` to make it obvious this is about the version and not an index? (Also applies to other similar channels, of course).

Reply:
Very good comment. I agree with you.

Reply:
Ok.

Reply:
Done